packages/eslint-plugin-react-hooks/__tests__/ESLintRuleExhaustiveDeps-test.js

```diff
@@ -5805,6 +5796,56 @@ const tests = {
         },
       ],
     },
+    {
+      code: normalizeIndent`	
+        function MyComponent(props) {	
+          let handleNext = () => {	
+            console.log('hello');	
+          };	
+          if (props.foo) {	
+            handleNext = () => {	
+              console.log('hello');	
+            };	
+          }	
+          useEffect(() => {	
+            return Store.subscribe(handleNext);	
+          }, [handleNext]);	
+        }	
```

Comment:
nit - there are tabs at the end of all of these lines

Reply:
https://github.com/facebook/react/pull/19590/commits/4dc35db7b77b345babba9146eb8ba40c2e2922f2